W-13072106-multiple-instances-note-kt (#1019)
mulesoft/docs-runtime-fabric: modules/ROOT/pages/install-multiple-instances.adoc

Installing multiple instances of Anypoint Runtime Fabric enables you to share the same cluster among multiple Runtime Fabrics, which helps you to use resources efficiently.

* To use this feature, you must upgrade to the minimum version of 2.2.5 of the Runtime Fabric agent that supports multiple instances.

> * Agent namespaces map one-to-one to app namespaces. Application namespaces cannot be shared amongst multiple instances of Runtime Fabric on the same cluster.

Reviewer: ```suggestion
* Agent namespaces map one-to-one to app namespaces. You cannot share app namespaces among multiple instances of Runtime Fabric on the same cluster.
```

> * There will be 1:1 mapping from the agent namespace to the app namespace. Application namespaces cannot be shared amongst multiple instances of Runtime Fabric on the same cluster.

Reviewer: Avoid future tense and "There is" construction. 

Agent namespaces map one-to-one to app namespaces. 
or 
For each agent namespace, you can have one app namespace.

Avoid passive voice: 
You cannot share app namespaces amongst multiple instances of Runtime Fabric on the same cluster.

* Hazelcast clustering is not yet supported with multiple instances.

[NOTE]

> For multiple instances, the first Runtime Fabric agent installation creates crd for persistencegateways.rtf.mulesoft.com and priorityclass rtf-components-high-priority resources. Though is not responsible for cleanup of these resources in case of uninstalling the Runtime Fabric agent.

Reviewer: Any font changes here?
Should "priorityclass" be two words?

Runtime Fabric agent doesn't clean up these resources when you uninstall the agent.
